.circleci/config.yml

```diff
@@ -70,7 +70,7 @@ jobs:
 
 workflows:
   version: 2
-  master:
+  main:
```

Comment:
I'm not sure how this key works in the yml file @julienw. Is this correct?

Reply:
Yeah, this is just an arbitrary name.
BTW: https://github.com/firefox-devtools/profiler-server/pull/109/files#diff-1d37e48f9ceff6d8030570cd36286a61R40 :)